### DISCUSSION TITLE
Tye Run in aspire

### DISCUSSION THREAD

**Discussion by kreditkenny** [microsoft/aspire#1931] [Q&A]
Hello,
We are just waking up now to realization that the tye project is no longer supported or will be evolving. 

We use .yml files and execute "tye run" in our development process. What is the equivalent in aspire?

**Comment by davidfowl** [comment]
`dotnet run` on the apphost project. The tye.yaml equivalent is the apphost.csproj and the program.cs file with logic to wire up infrastructure. You need to install the aspire workload.

https://learn.microsoft.com/en-us/dotnet/aspire/get-started/aspire-overview
